caluma/caluma_analytics/filters.py

```diff
@@ -53,7 +50,7 @@ class AnalyticsTableOrderSet(FilterSet):
     meta = MetaFieldOrdering()
     attribute = AttributeOrderingFactory(
         models.AnalyticsTable,
-        fields=["created_at", "modified_at", "slug", "name"],
+        fields=["created_at", "modified_at", "slug", "name", "description"],
```

Comment:
Does it make sense to allow ordering by description? No need to fix, I just don't think it's particularly useful

Reply:
Sorry, should have marked as `draft`. Wanted to first fix CI, that's why I pushed early, even before I took a closer look at the changes :sweat_smile: